ipywidgets/widgets/widget_selection.py

```diff
@@ -384,9 +385,22 @@ class ToggleButtonsStyle(DescriptionStyle, CoreWidget):
     button_width: str
         The width of each button. This should be a valid CSS
         width, e.g. '10px' or '5em'.
+
+    button_color: str or None
+        The color of each button. This should be a valid CSS color unit,
+        e.g. 'yellow', '#FFFF00', '255,255,0'.
```

Comment:
The rgb color should be of the form `rgb(255,255,0)`, right? https://developer.mozilla.org/en-US/docs/Web/CSS/color

Reply:
Yes, you are totally right. Thanks for the catch! I will make updates accordingly.